modules/openapi-generator/src/main/resources/rust/reqwest/api.mustache

```diff
@@ -78,15 +78,22 @@ impl {{{classname}}} for {{{classname}}}Client {
         {{/isKeyInHeader}}
         {{/isApiKey}}
         {{#isBasic}}
+        {{^isBasicBearer}}
         if let Some(ref auth_conf) = configuration.basic_auth {
             req_builder = req_builder.basic_auth(auth_conf.0.to_owned(), auth_conf.1.to_owned());
         };
+        {{/isBasicBearer}}
         {{/isBasic}}
         {{#isOAuth}}
         if let Some(ref token) = configuration.oauth_access_token {
             req_builder = req_builder.bearer_auth(token.to_owned());
         };
         {{/isOAuth}}
+        {{#isBasicBearer}}
```

Comment:
@boxdot thanks for the PR. I would suggest put this new block inside the {{#isBasic}} ... {{/isBasic}} block. What do you think?

And what about creating bearer_access_token instead or reusing oauth_access_token?

Some other clients are using `access_token` so it can be shared with OAuth, Bearer authentications.

Reply:
(I'm working with @boxdot)

I think having a single `access_token` also works, but I guess by adding `bearer_access_token` (see last commit in PR), we avoid breaking existing clients.